src/box/xrow_update_field.h

```diff
@@ -99,6 +99,38 @@ struct xrow_update_field;
 struct xrow_update_op;
 struct tuple_dictionary;
 
+/**
+ * Scalar type.
+ */
+enum xrow_update_scalar_type {
+	XUPDATE_TYPE_NONE = 0, /** UNINITIALIZED */
+	XUPDATE_TYPE_DECIMAL = 1, /** decimal_t */
+	XUPDATE_TYPE_DOUBLE = 2, /** double */
+	XUPDATE_TYPE_FLOAT = 3, /** float */
+	XUPDATE_TYPE_INT = 4, /** struct int96_num */
+	XUPDATE_TYPE_STR = 5, /** xrow_update_string_rope */
```

Comment:
We don't put comments on the same line as code (you might see violations in the old code, but it is legacy), lets move them above each of the constants. Or just drop them, they are obvious anyway.

Reply:
Fixed to separate line for the comment.